### DISCUSSION TITLE
weekly/2023-12-24

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#159] [Announcements]
# weekly/2023-12-24

① 掌中宇宙，浩瀚星海，一眼万年 ② Vue 2 即将停止维护 ③ Emoji 搜索引擎 ④ JS 引擎和运行时合集 ⑤ Bun 2024年目标 ⑥ GitHubUnwrapped - 2023年的GitHub行为总结 ⑦ Devv AI 是如何构建高效的 RAG 系统的

https://sugarat.top/weekly/2023-12-24.html

**Comment by ttmars** [comment]
666

**Comment by Perfecto23** [comment]
MDH偶尔会断更，前端食堂质量不高，基本都是其他周刊的重复内容，你这个质量很高，我一直在追

**Reply by ATQQ** [reply]
感谢支持

我这边内容渠道也是 各大周刊(下面的 https://www.fre123.com/weekly 有大部分周刊的汇总 ) + X + 微信群

然后从里面挑些轻松愉快的分享一下😋
